source/_integrations/homekit.markdown

```diff
@@ -458,6 +458,10 @@ automation:
     packet: XXXXXXXX
 ```
 
+## Events
+
+This integration emits events as `homekit_state_change` on the event bus. This can be used for example in automations to known whether an entity's state got changed from HomeKit specifically. 
```

Comment:
Maybe an example event would be useful here?

Reply:
Unresolving this comment.

@parautenbach You marked this comment as resolved without a response or other actions taken. Care to share why you marked it resolved?

Reply:
It must've been by accident, sorry. I have every intention to add an example. I thought of it initially, but I'm not sure how complete examples should be. Dealing directly with the event bus almost allows one to do anything, so I was thinking of just having the trigger part of an automation as an example. Anyway, I'll try to have a look tonight (I forgot about this a little after seeing the notification) and then you guys can tell me whether it's good or not! :-)